tools/publish/run-processor.sh

```
@@ -1,7 +1,11 @@
+CURRENT_DIR="$(dirname "$0")"
+export CONFIG_NAME=prodtest
+export THECOIN_SECRETS=${CURRENT_DIR}/env
 
-SCRIPT_DIR="$(dirname "$0")"
-CURRENT_DEPLOY_PATH=$(cat "$SCRIPT_DIR/current_deploy.txt")
-echo "Running processor on $(date) for version: $CURRENT_DEPLOY_PATH"
-
-cd $CURRENT_DEPLOY_PATH/apps/tx-processor
-yarn prod:test
\ No newline at end of file
+if docker compose --ansi never -f ${CURRENT_DIR}/docker-compose.yaml up; then
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)